ipaplatform/fedora/constants.py

```diff
@@ -22,6 +28,9 @@ class FedoraConstantsNamespace(RedHatConstantsNamespace):
     # secure remote password, and DSA cert authentication.
     # see https://fedoraproject.org/wiki/Changes/CryptoPolicy
     TLS_HIGH_CIPHERS = "PROFILE=SYSTEM:!3DES:!PSK:!SRP:!aDSS"
-
+    if not HAS_NFS_CONF:
+        SECURE_NFS_VAR = "SECURE_NFS"
+    else:
+        SECURE_NFS_VAR = None
```

Comment:
base/constants.py will define this as "SECURE_NFS" by default so I think you can make this:
if HAS_NFS_CONF:
    SECURE_NFS_VAR = None